assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/components/table/table.tsx

```diff
@@ -112,13 +112,13 @@ export const Table = ({ propertiesTableTab }: ITableProps): React.JSX.Element =>
         return (
           <div className={ 'properties-table--actions-column' }>
             {
-              info.row.original.type === 'document' &&
-              info.row.original.data !== null &&
+              (info.row.original.type === 'document' || info.row.original.type === 'asset') &&
```

Comment:
The same will also later apply for the objects. I guess, while we already working on it - let's add the objects, too.